tests/test_klaviyo_discovery.py

```diff
@@ -63,6 +67,15 @@ def test_run(self):
                     item.get("breadcrumb", ["properties", None])[1] for item in metadata
                     if item.get("metadata").get("inclusion") == "automatic"
                 )
+                actual_replication_keys = set(
+                    stream_properties[0].get(
+                        "metadata", {self.REPLICATION_KEYS: []}).get(self.REPLICATION_KEYS, [])
+                )
+                
+                # verify there are no duplicate metadata entries
+                for md in metadata:
+                    self.assertEqual(metadata.count(md),1,msg="There is duplicated metadata in '{}' stream".format(stream))
```

Comment:
```suggestion
                actual_fields = []
                for md_entry in metadata:
                    if md_entry['breadcrumb'] != []:
                        actual_fields.append(md_entry['breadcrumb'][1])

                # Verify there are no duplicate metadata entries
                self.assertEqual(len(actual_fields), len(set(actual_fields)), msg = "duplicates in the metadata entries retrieved")
```
Update the above assertion to keep consistency with other taps.